pkg/apiserver/filters/auditing.go

```diff
@@ -0,0 +1,42 @@
+package filters
+
+import (
+	"k8s.io/klog"
+	"kubesphere.io/kubesphere/pkg/apiserver/auditing"
+	"kubesphere.io/kubesphere/pkg/apiserver/request"
+	"net/http"
+)
+
+func WithAuditing(handler http.Handler, a auditing.Auditing) http.Handler {
+
```

Comment:
Move code up here, we can do shirt circuit, return fast.
```suggestion
if a != nil && !a.Enable() {
  return handler
}
```

Reply:
Auditing level can be modified with cr kube-auditing-webhook, so it need to judge every time.